### PR TITLE
Add docs and code for sending binary SMS

### PR DESCRIPTION
**Pull request by didier-wg2**

### CODE REVIEW

File: docs/sms-send-how-to.md [hunk 1]
```diff
@@ -3,20 +3,22 @@ title: Send SMS
...
-code: https://github.com/working-group-two/docs.wgtwo.com/blob/master/examples/sms/src/main/kotlin/SendSms.kt
+code: https://github.com/working-group-two/docs.wgtwo.com/blob/master/examples/sms/src/main/kotlin/SendTextSms.kt
 ---
 
 # How to send SMS
 
 ## Overview
 
-This example shows how you can send SMSes. In order to send SMSes your credentials need to have the
+This example shows how you can send text SMSes. In order to send SMSes your credentials need to have the
```
Comment (david-wg2) [inline docs/sms-send-how-to.md]: Doesn't it show how to send "text and binary SMSes" ?

File: docs/sms-send-how-to.md [hunk 2]
```diff
@@ -42,6 +44,10 @@ grpcurl \
   messaging.MessageCore.SendTextMessage
 ```
 
+### Send binary messages
+
+Unfortunately, `grpcurl` only allows to send JSON formatted strings, so it can't be used to send binary SMS.
```
Comment (david-wg2) [inline docs/sms-send-how-to.md]: Maybe add a "please have a look at the code example below" or something ?

File: docs/sms-send-how-to.md [hunk 3]
```diff
@@ -42,6 +44,10 @@ grpcurl \
   messaging.MessageCore.SendTextMessage
 ```
 
+### Send binary messages
```
Comment (david-wg2) [inline docs/sms-send-how-to.md]: Since the previous heading is `Send text SMS`, this should be `Send binary SMS`

File: docs/sms-send-how-to.md [hunk 4]
```diff
@@ -51,9 +57,12 @@ Then you can add `messaging-grpc` and `utils-grpc`:
...
 
-### Send SMS
+### Send text SMS
 <GithubCode :to="$frontmatter.code" />
 
+### Send binary SMS
+<GithubCode :to="https://github.com/working-group-two/docs.wgtwo.com/blob/master/examples/sms/src/main/kotlin/SendBinarySms.kt" />
+
 ## Resources
 * [SendSmsDemo.kt](https://github.com/working-group-two/wgtwo-kotlin-code-snippets/blob/master/src/main/kotlin/com/wgtwo/example/sendsms/SendSmsDemo.kt)
```
Comment (david-wg2) [inline docs/sms-send-how-to.md]: Not really part of the change, but remove this?

Comment (didier-wg2) [reply docs/sms-send-how-to.md]: Done

File: docs/sms-send-how-to.md (hunk 3 above)
Comment (didier-wg2) [reply docs/sms-send-how-to.md]: Done

File: docs/sms-send-how-to.md (hunk 2 above)
Comment (didier-wg2) [reply docs/sms-send-how-to.md]: Done

File: docs/sms-send-how-to.md (hunk 1 above)
Comment (didier-wg2) [reply docs/sms-send-how-to.md]: Done
